src/main/java/az/edu/turing/exception/AlreadyExistsException.java

```diff
@@ -0,0 +1,10 @@
+package az.edu.turing.exception;
+
+public class AlreadyExistsException extends RuntimeException {
+
+    public String message;
```

Comment:
We don't need a message field,There is  a super(message) const inside  RuntimeException